pwnlib/tubes/ssh.py

```diff
@@ -1031,7 +1048,29 @@ def is_exe(path):
 %(func_src)s
 %(func_name)s(*%(func_args)r)
 
-os.execve(exe, argv, env)
+# os.execve does not allow us to pass empty argv[0]
+# Therefore we use ctypes to call execve directly
+# os.execve(exe, argv, env)
+        
+# Transform envp from dict to list
+env_list = [key + b"=" + value for key, value in env.items()]
```

Comment:
Please [add a check](https://github.com/python/cpython/blob/025995feadaeebeef5d808f2564f0fd65b704ea5/Modules/posixmodule.c#L6476) that `key` does not contain `=`.